tests/foreman/api/test_docker.py

```diff
@@ -1132,7 +1132,7 @@ def setUpClass(cls):
         cls.cr_external = entities.DockerComputeResource(
             name=gen_string('alpha'),
             organization=[cls.org],
-            url=settings.docker.external_url,
+            url='http://{0}:2375'.format(settings.server.hostname),
```

Comment:
1) We can move that `'http://{0}:2375'` part to a constants as we did with Libvirt URL.
2) With that change property `settings.docker.external_url` seems as redundant one which is basically not good idea from logical point of view. Can we make `'http://{0}:2375'.format(settings.server.hostname)` as a default one value for `settings.docker.external_url` and leave possibility for user to specify his own URL in case he wants to do that?